minard/templates/pca_set.html

```diff
@@ -0,0 +1,45 @@
+{% extends "layout.html" %}
+{% block title %}PCA Set{% endblock %}
+{% block head %}
+  {{ super() }}
+{% endblock %}
+{% block body %}
+  {{ super() }}
+
+  <div class="container">
+    <div class="row">
+      <div class="col-md-12">
+          <h2 style="text-align: center">TELLIE PCA dataset: {{ data["name"] }}</h2>
+            <table class="table table-bordered table-condensed table-striped">
+              <thead>
+                <th style="text-align: center; vertical-align: middle;">Fibre</th>
+                <th style="text-align: center; vertical-align: middle;">Run</th>
+                <th style="text-align: center; vertical-align: middle;">Validation 1</th>
```

Comment:
What do the validation 1 and validation 2 numbers mean? Are they meaningful to display?

Reply:
They are results of distinguished RAT Processors, each looking at the same run but Val1 with 'raw' and Val2 with 'corrected' hits. As such, the number is important.

More details:
[https://www.snolab.ca/snoplus/private/DocDB/cgi/ShowDocument?docid=7699](https://www.snolab.ca/snoplus/private/DocDB/cgi/ShowDocument?docid=7699)
[https://www.snolab.ca/snoplus/private/DocDB/cgi/ShowDocument?docid=7640](https://www.snolab.ca/snoplus/private/DocDB/cgi/ShowDocument?docid=7640)